src/test/java/quickndirty/minisurveymonkey/WebControllerTest.java

```diff
@@ -0,0 +1,27 @@
+package quickndirty.minisurveymonkey;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class WebControllerTest {
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void checkCreateSurveyPageStatus() throws Exception {
```

Comment:
Nice. @nikolaerak, is this what you were think about for mvc tests as well. We could definitely do more stuff like this.